parser/prism/Translator.cc

```diff
@@ -476,6 +476,37 @@ std::unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::Rational>(parser.translateLocation(loc), value);
         }
+        case PM_REGULAR_EXPRESSION_NODE: {
+            auto regularExpressionNode = reinterpret_cast<pm_regular_expression_node *>(node);
+            pm_location_t *loc = &regularExpressionNode->base.location;
+            pm_location_t *closingLoc = &regularExpressionNode->closing_loc;
+
+            // Sorbet represents the regex content as a vector of string nodes
+            parser::NodeVec parts;
+
+            auto source = parser.extractString(&regularExpressionNode->unescaped);
+            if (!source.empty()) {
+                auto sourceStringNode =
+                    make_unique<parser::String>(parser.translateLocation(loc), gs.enterNameUTF8(source));
+                parts.emplace_back(std::move(sourceStringNode));
+            }
+
+            std::string_view optString;
+
+            auto optStart = closingLoc->start + 1; // one character after the closing `/`
+            auto optEnd = closingLoc->end;
+            auto optLength = optEnd - optStart;
+
+            // Some regexps have options, like `/foo/i`
+            // Some have no options
+            if (optLength > 0) {
```

Comment:
Nit: wording is a bit awkward ("some have" implies "some have no")

```suggestion
            if (optLength > 0) { // Optional options like the `i` in `/foo/i`
```